### PR TITLE
Prep for sampling

### PR DESCRIPTION
**Pull request by cremebrule**

### CODE REVIEW

File: omnigibson/prims/entity_prim.py [hunk 1]
```diff
@@ -1229,8 +1229,7 @@ def kinematic_only(self, val):
                 body's pose / velocities. See https://docs.omniverse.nvidia.com/app_create/prod_extensions/ext_physics/rigid-bodies.html?highlight=rigid%20body%20enabled#kinematic-rigid-bodies
                 for more information
         """
-        for link in self.links.values():
-            link.kinematic_only = val
+        self.root_link.kinematic_only = val
```
Comment (ChengshuLi) [inline omnigibson/prims/entity_prim.py:1232]: Is this intended?

File: omnigibson/scenes/scene_base.py [hunk 2]
```diff
@@ -398,8 +398,8 @@ def add_object(self, obj, register=True, _is_call_from_simulator=False):
         # let scene._load() load the object when called later on.
         prim = obj.load()
 
-        # If this object is fixed, disable collisions between the fixed links of the fixed objects
-        if obj.fixed_base:
+        # If this object is fixed and is NOT an agent, disable collisions between the fixed links of the fixed objects
```
Comment (ChengshuLi) [inline omnigibson/scenes/scene_base.py:401]: Maybe add a comment about Tiago specifically?

**Review by ChengshuLi — APPROVED** [review]

File: omnigibson/prims/entity_prim.py (hunk 1 above)
Comment (cremebrule) [reply omnigibson/prims/entity_prim.py:1232]: Yeah -- this is to remove all those joint errors you were getting

File: omnigibson/scenes/scene_base.py (hunk 2 above)
Comment (cremebrule) [reply omnigibson/scenes/scene_base.py:401]: done!

**Pull request merged**